.github/workflows/libvmaf.yml

```diff
@@ -75,11 +67,33 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run meson
         run: |
-          #sudo chown -R runner: $HOME/.ccache
-          meson setup libvmaf libvmaf/build --buildtype release
+          meson setup libvmaf libvmaf/build --buildtype release --prefix $(pwd)/install
```

Comment:
```suggestion
          meson setup libvmaf libvmaf/build --buildtype release --prefix $PWD/install
```
`$PWD` should theoretically give the same thing as running the `pwd` command

Reply:
Cool, thanks.